tests/test_modeling_albert.py

```diff
@@ -227,6 +230,20 @@ class AlbertModelTest(ModelTesterMixin, unittest.TestCase):
         else ()
     )
 
+    # special case for ForPreTraining model
+    def _prepare_for_class(self, inputs_dict, model_class, return_labels=False):
```

Comment:
some models have this function added to their `...ModelTest`. It's necessary to make the `test_training` fn pass for all models.